src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -0,0 +1,315 @@
+                this.state.metric,
+                this.state.threshold
+              ),
+            ]
+          : undefined,
+      actions: [{id: NOTIFY_EVENT_ACTION}],
+      actionMatch: 'all',
+      frequency: 5,
+    };
+  }
+
+  setStateAndUpdateParents<K extends keyof State>(
+    state:
+      | ((
+          prevState: Readonly<State>,
+          props: Readonly<Props>
+        ) => Pick<State, K> | State | null)
+      | Pick<State, K>
+      | State
+      | null,
+    callback?: () => void
+  ): void {
+    this.setState(state, () => {
+      callback?.();
+      this.props.onChange(this.getUpdatedData());
+    });
+  }
+
+  getEndpoints(): [string, string][] {
+    return [['conditions', `/projects/${this.props.organization.slug}/rule-conditions/`]];
+  }
+
+  onLoadAllEndpointsSuccess(): void {
+    const conditions = this.state.conditions?.filter?.(object =>
+      Object.values(METRIC_CONDITION_MAP).includes(object?.id)
+    );
+
+    if (!conditions || conditions.length === 0) {
+      this.setStateAndUpdateParents({conditions: undefined});
+      return;
+    }
+
+    const {intervalChoices, placeholder, interval} = unpackConditions(conditions);
+    if (!intervalChoices || !placeholder || !interval) {
+      Sentry.withScope(scope => {
+        scope.setExtra('props', this.props);
+        scope.setExtra('state', this.state);
+        Sentry.captureException(
+          Error(
+            'Interval choices or value placeholder sent from API endpoint is inconsistent or empty'
+          )
+        );
+      });
+      this.setStateAndUpdateParents({conditions: undefined});
+      return;
+    }
+
+    this.setStateAndUpdateParents({
+      conditions,
+      intervalChoices,
+      placeholder,
+      interval,
+    });
+  }
+
+  renderBody(): React.ReactElement {
+    const issueAlertOptionsChoices = this.getIssueAlertsChoices(
+      this.state.conditions?.length > 0
+    );
+    return (
+      <React.Fragment>
+        <PageHeadingWithTopMargins withMargins>
+          {t('Create an alert')}
+        </PageHeadingWithTopMargins>
+        <RadioGroupWithPadding
+          choices={issueAlertOptionsChoices}
+          label={t('Options for creating an alert')}
+          onChange={alertSetting => this.setStateAndUpdateParents({alertSetting})}
+          value={this.state.alertSetting}
+        />
+      </React.Fragment>
+    );
+  }
+}
+
+export default withOrganization(IssueAlertOptions);
+
+const CustomizeAlertsGrid = styled('div')`
+  display: grid;
+  grid-template-columns: repeat(5, max-content);
+  grid-gap: 10px;
+  align-items: center;
+`;
+
+const InlineInput = styled(Input)`
+  width: 80px;
+`;
+
+const InlineSelectControl = styled(SelectControl)`
+  width: 160px;
+`;
+const RadioGroupWithPadding = styled(RadioGroup)`
+  padding: ${space(3)} 0;
+  margin-bottom: 50px;
+  box-shadow: 0 -1px 0 rgba(0, 0, 0, 0.1);
+`;
+const PageHeadingWithTopMargins = styled(PageHeading)`
+  margin-top: 65px !important;
```

Comment:
I based the margin size off the mocks.
We could also just not have margins. I think it looks ok.

Reply:
1. We could use PageHeading without specifying `WithMargins` and provide the margins directly.
2.  Allow the the top margin to be specified as an argument in PageHeading.

Went for this because I wanted the bottom margin to be consistent with PageHeading and didn't want to make the component more complex.

Reply:
Settled offline, `!important` was redundant due to styled.